api/src/models/JobMetadata.ts

```diff
@@ -1,12 +1,16 @@
 import PageOrder from './PageOrder';
 import DocumentOrder from './DocumentOrder';
+import AudioOrder from './AudioOrder';
 import Job from './Job';
 
 class JobMetadata {
     job: Job;
+    _filename: string;
```

Comment:
This property currently does not appear to be used. It looks like the Ruby code uses the filename property in both the constructor and the save method. Maybe you should make some adjustments here to match.